### PR TITLE
EL.wallet-fix

### PR DESCRIPTION
**Pull request by emlowe**
Various updates
Update mypy to 1.11.1 and make sure pre-commit runs it correctly - fix issues with mypy
add prettier to pre-commit
run pre-commit to get isort/black/flake8 updates - this does need to be added to CI - looks like we forgot to do that

fixes needed for compatibility with chia-blockchain 2.4.4

### CODE REVIEW

**Comment by emlowe** [comment]
@SocketSecurity ignore pypi/cfgv@3.4.0, pypi/pre-commit@3.7.1, pypi/pyupgrade@3.16.0, pypi/identify@2.6.1, pypi/types-pyyaml@6.0.12.20240917

File: README.md [hunk 1]
```diff
@@ -5,49 +5,49 @@
...
-  * Port: 31999
-  * Application Name: Only available from source builds
-
-When compiling from source, the "mode" is controlled by the `.env` file.  Each application, or mode, is offered as precompiled binaries, appropriate for most users.
+- **Chia Climate Tokenization**:
+  - Mode: Registry
+  - Use case: A registry would use this to tokenize carbon credits onto the Chia blockchain
+  - Port: 31312
+  - Application Name: climate-tokenization-chia
+  - _Only listens on localhost for connections from the [Climate Tokenization Engine](https://github.com/Chia-Network/Climate-Tokenization-Engine)_
```
Comment (Quexington) [inline README.md:13]:
```suggestion
  - Only listens on localhost for connections from the [Climate Tokenization Engine](https://github.com/Chia-Network/Climate-Tokenization-Engine)_
```

File: app/api/v1/activities.py [hunk 2]
```diff
@@ -162,13 +162,13 @@ async def get_activity_by_cw_unit_id(
...
     else:
-        logger.warning(f"retrieved unit does not contain marketplace identifier. unable to get activity record")
+        logger.warning("retrieved unit does not contain marketplace identifier. unable to get activity record")
         return schemas.ActivityRecordResponse()
 
     activity_filters["and"].append(models.Activity.mode == action_mode)
     activity_filters["and"].append(models.Activity.coin_id == coin_id)
 
-    activities: [models.Activity]
+    activities = [models.Activity]
```
Comment (Quexington) [inline app/api/v1/activities.py:171]: this... worked before?

File: app/crud/chia.py [hunk 3]
```diff
@@ -46,7 +46,7 @@ def _get_paginated_data(self, path: str, search_params: Dict[str, Any]) -> List[
         Returns:
             A list of all data retrieved from the paginated API.
         """
-        all_data = []
+        all_data: list[Any] = []
```
Comment (Quexington) [inline app/crud/chia.py:49]: list[dict[str, Any]] it seems like might be right?

**Review by Quexington — CHANGES_REQUESTED** [review]
Very silly review

File: app/api/v1/activities.py (hunk 2 above)
Comment (emlowe) [reply app/api/v1/activities.py:171]: it did something probably :-) 🤷

File: app/crud/chia.py (hunk 3 above)
Comment (emlowe) [reply app/crud/chia.py]: Yea, changed

**Review by Quexington — APPROVED** [review]

File: app/api/v1/activities.py (hunk 2 above)
Comment (wwills2) [reply app/api/v1/activities.py:171]: i was under the impression that such a line did quite literally nothing (in the sense that it resulted in an unusable variable until initialization), but i didn't want to cause a headache for myself if the author of line 80 knew something i didn't :)

**Review by wwills2 — APPROVED** [review]
approving specifically the changes made to files I recently edited

**Pull request merged**